src/redux/reducers/beersReducer.js

```diff
@@ -15,14 +37,19 @@ export default function beersList(state = initialState, action) {
       };
     }
     case GET_BEER_LIST_SUCCESS: {
-      const { beersList: oldBeers } = state;
+      const { beersList: oldBeers, params: oldParams } = state;
       const {
         payload: { beers },
       } = action;
 
+      const filteredBeers = beers.filter(
+        (loadedBeer) => oldBeers.findIndex((beer) => loadedBeer.id === beer.id) === -1
```

Comment:
I guess you can use 'find' method, which will return undefined if item is not found
And return boolean of find result as a filtering result
something like:
`found = oldBeers.find(beer => loadedBeer.id === beer.id);
  return Boolean(found)`